assets/_scss/themes/_usa.scss

```diff
@@ -16,13 +17,13 @@ li {
 }
 
 a {
-  color: $color-main;
+  color: $color-primary;
```

Comment:
Good point and this was previously in variables, but just removed to put it in theme instead. I don't have an answer yet about what style should be considered a theme or variable, but we should think more about it.